de.renew.Renew.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>de.renew.Renew</id>
+  <name>Renew: The Reference Net Workshop</name>
```

Comment:
```suggestion
  <name>Renew</name>
```

Reply:
Changed the name according to your suggestion.